builder/dockerfile/internals.go

```diff
@@ -123,8 +122,7 @@ func (b *Builder) runContextCommand(args []string, allowRemote bool, allowLocalD
 				return err
 			}
 			defer os.RemoveAll(filepath.Dir(fi.Path()))
-			decompress = false
-			infos = append(infos, copyInfo{fi, decompress})
+			infos = append(infos, copyInfo{fi, false})
```

Comment:
It's a nit so feel free to ignore, but I think it would be cool to have it like 

```go
infos = append(infos, copyInfo{
  FileInfo: fi,
  decompress: false,
})
```

to know what the `false` mean.